### ISSUE TITLE
Expected server HTML to contain a matching <main> in <div>

### ISSUE THREAD

**Issue by adriatic** [redwoodjs/redwood#1934]
The title (Expected server HTML to contain a matching ...) is the error message that shows after executing the very first command `yarn create redwood-app./appname`. Note that this message with the stacktrace is shown in the browser console:

![image](https://user-images.githubusercontent.com/2712405/110176912-11ca3200-7dd2-11eb-96b2-44e421de5b5a.png)

---

**Copy of the text on the above image:**

```
index.js:1 Warning: Expected server HTML to contain a matching <main> in <div>.
    in main (created by SplashPage)
    in SplashPage (created by RouterImpl)
    in RouterImpl (created by LocationProvider)
    in LocationProvider (created by Context.Consumer)
    in Location (created by Router)
    in Router (created by Routes)
    in Routes (created by App)
    in FlashProvider (created by RedwoodApolloProvider)
    in GraphQLHooksProvider (created by RedwoodApolloProvider)
    in ApolloProvider (created by ApolloProviderWithFetchConfig)
    in ApolloProviderWithFetchConfig (created by RedwoodApolloProvider)
    in FetchConfigProvider (created by RedwoodApolloProvider)
    in RedwoodApolloProvider (created by App)
    in FatalErrorBoundary (created by App)
    in App
```

**Comment by peterp** [comment]
@thedavidprice we should definitely fix this before the next release

**Comment by dthyresson** [comment]
Should the copy reference:

`Thanks for choosing Redwood Router!` 

And instead just RedwoodJS -- since router is no longer to be a standalone package?

**Comment by dthyresson** [comment]
The markup looks ok (a single `main` element with closing tag) in https://github.com/redwoodjs/redwood/blob/a1c4f38dd9bff76a29c3d0415e53f621bcec38d6/packages/router/src/splash-page.tsx#L2 

but might there be something with prerender that encloses this differently now?

**Comment by thedavidprice** [comment]
@dac09 I took a look but couldn't see anything obvious. Assuming there's something deeper in prerender world causing this. Can you dig into it?

**Comment by thedavidprice** [comment]
closing this out in favor of https://github.com/redwoodjs/redwood/issues/1887

Expecting https://github.com/redwoodjs/redwood/pull/2056 fix to be included in upcoming v0.28 release
